styles.css

```diff
@@ -0,0 +1,190 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    user-select: none;
+}
+body{
+    font-family: 'Epilogue', sans-serif;
+    font-size: 18px;
+    background-color:hsl(0, 0%, 98%);
+}
+.navbarContainer{
+    display: flex;
+    flex-direction: row;
+    margin: 1rem 2rem;
+    align-items: center;
+}
+.navbarContainer img{
+    display:none;
+}
+.navbarMenu{
+    display:flex;
+    flex-direction: row;
+    align-items: center;
+    width:100%;
+    justify-content: space-between;
+}
+.navbarItems{
+    list-style: none;
+}
+.navbarItems img{
+    vertical-align: middle;
+}
+.navbarItems li{
+    float:left;
+    padding:1rem;
+    color:hsl(0, 0%, 41%);
+}
+.navbarItems li:hover{
+    cursor:pointer;
+    color:black;
+}
+.navbarItems img{
+    display:inline-block;
+}
+.authOptions{
+    display:flex;
+    flex-direction: row;
+    gap:3rem;
+}
+.login{
+    padding:0.5rem 1rem;
+}
+.login:hover{
+    cursor:pointer;
+}
+.register{
+    border:1px black solid;
+    border-radius: 0.5rem;
+    padding:0.5rem 1rem;
+}
+.register:hover{
```

Comment:
możesz to zapisać w jednym selektorze z 'login:hover':
```
.login:hover,
.register:hover{
    cursor:pointer;
}
```
albo, co byłoby jeszcze lepsze z punktu widzenia zasady DRY (dont repeat yourself) - stworzenie jednej klasy, która to uzwględni, czyli np. 'authAction' i dodanie tej klasy do obu miejsc w HTML - loginu i register